config/locales/en/activemodel.yml

```diff
@@ -210,6 +213,16 @@ en:
           attributes:
             attachment_type:
               invalid: "'%{attachment_type}' is invalid"
+        bank_statement:
+          attributes:
+            original_file:
+              content_type_invalid: "%{file_name} must be a DOC, DOCX, RTF, ODT, JPG, BMP, PNG, TIF or PDF"
```

Comment:
I am assuming this is placeholder at the moment as we have not decided what can and can't be uploaded for this page. When this is decided we will need to make the changes in dropzone.js `app/webpack/src/dropzone.js:9`, so that the type of file allowed is dependent on the page the user is on

Reply:
we should add a CSV at least; consider restricting to remove doc docx and rtf; we will need to differentiate dropzone for this particular page in this case as well as validators

Reply:
should probably do this in future iterations once we have the requirements clarified as it can be done retrospectively

Reply:
separate ticket https://dsdmoj.atlassian.net/browse/AP-3281